guides/common/modules/proc_synchronizing-smart-proxy-directly-from-red-hat-cdn.adoc

```diff
@@ -0,0 +1,16 @@
+[id="Synchronizing_{smart-proxy-context}_Directly_From_Red_Hat_CDN_{context}"]
+= Synchronizing {SmartProxy} Directly From Red Hat CDN
+
+You can use simplified alternate content sources to configure your {SmartProxy} to sync content directly from Red Hat CDN.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Alternate Content Sources*.
+. Click *Add Source* and set the *Source type* as *Simplified*.
+. Set the *Content type* to *Yum*.
+. In the *Name* field, enter a name for the alternate content source.
+. Optional: In the *Description* field, provide a description for the alternate content source.
+. Select {SmartProxies} that you want to sync directly from Red Hat CDN.
+. Optional: Select *Use HTTP proxies* if you want the ACS to use the {SmartProxyServer}’s HTTP proxy.
+. Select the Red Hat products that should be synced to the {SmartProxy} from Red Hat CDN.
+. Review details and click *Add*.
+. Navigate to *Content* > *Alternate Content Sources*, click the vertical ellipsis next to the newly created alternate content source, and select *Refresh*.
```

Comment:
I realize users may think that the refresh will download the content for them, especially since refreshing is a new concept. What do you think about adding a line like:

```suggestion
. Navigate to *Content* > *Alternate Content Sources*, click the vertical ellipsis next to the newly created alternate content source, and select *Refresh*.
. Now, whenever the {SmartProxy} downloads content, it will come from the CDN rather than from the Satellite.
```

Reply:
@maximiliankolb can we add a line at the end to this effect? If yes, should it be outside the procedure section?

Reply:
Sure, adding something below the procedure is possible.